src/prommis/uky/uky_flowsheet.py

```diff
@@ -861,5 +873,956 @@ def display_results(m):
+    m.fs.reep_pump.feed_rate.fix()
+    m.fs.reep_pump.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": reep_pump_accounts,
+            "scaled_param": m.fs.reep_pump.feed_rate,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 9.5 is UKy Rare Earth Element Precipitation - Filter Press
+    reep_filter_press_accounts = ["9.5"]
+    m.fs.reep_filter_press = UnitModelBlock()
+    m.fs.reep_filter_press.volume = Var(initialize=0.405, units=units.ft**3)
+    m.fs.reep_filter_press.volume.fix()
+    m.fs.reep_filter_press.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": reep_filter_press_accounts,
+            "scaled_param": m.fs.reep_filter_press.volume,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 9.8 is UKy Rare Earth Element Precipitation - Roaster
+    reep_roaster_accounts = ["9.8"]
+    m.fs.reep_roaster = UnitModelBlock()
+    m.fs.reep_roaster.duty = Var(initialize=0.035, units=units.MBTU / units.hr)
+    m.fs.reep_roaster.duty.fix()
+    m.fs.reep_roaster.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": reep_roaster_accounts,
+            "scaled_param": m.fs.reep_roaster.duty,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # Roasting costs
+    # 3.1 is UKy Roasting - Storage Bins
+    R_storage_bins_accounts = ["3.1"]
+    m.fs.R_storage_bins = UnitModelBlock()
+    m.fs.R_storage_bins.capacity = Var(initialize=10.0, units=units.ton)
+    m.fs.R_storage_bins.capacity.fix()
+    m.fs.R_storage_bins.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": R_storage_bins_accounts,
+            "scaled_param": m.fs.R_storage_bins.capacity,
+            "source": 1,
+            "n_equip": 2,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 3.2 is UKy Roasting - Conveyors
+    R_conveyors_accounts = ["3.2"]
+    m.fs.R_conveyors = UnitModelBlock()
+
+    flow_3_2 = value(
+        units.convert(
+            flowsheet.fs.roaster.flow_mas_product[0]
```

Comment:
There appears to be a typo in the two roaster flowrates, and I'm surprised that this doesn't fail the tests.

Reply:
This is the correct name of the unit property, but the unit property itself has this typo. I'll try to fix it now - it shouldn't take long.

Reply:
Fixed.